backend/app/services/learning-service.py

```
@@ -0,0 +1,110 @@
+from typing import Dict, List, Any, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)